### ISSUE TITLE
Binding error in console when using a number as the text property for a Span

### ISSUE THREAD

**Issue by andreacappadona17** [NativeScript/NativeScript#8647]
**Environment**
Nativescript Core Typescript
 - CLI: 6.7.4
 - Cross-platform modules: 6.5.6
 - Android Runtime: 6.5.1
 - iOS Runtime: 6.5.1
 - Plugin(s):

**Describe the bug**
If you have a view with something like this:
```
<Label textWrap="true">
  <FormattedString>
    <Span text="ID: "></Span>
    <Span text="{{ id }}"></Span>
  </FormattedString>
</Label>
```
And _id_ was a number, in the bindingContext, the number would not be showing in the view, and there would be some binding errors reported in the console.

**Expected behavior**
The label should be showing and there should be no errors in the console.

**Additional context**
The problem is probably related to this line
https://github.com/NativeScript/NativeScript/blob/29541b89b17ed001ea7a93a9860405e330f92734/nativescript-core/ui/text-base/span.ts#L68
There should be a check for the replace function, as numbers to not have it.

`this._text = value && value.replace ? value.replace("\\n", "\n").replace("\\t", "\t") : value;`

**Comment by NathanWalker** [comment]
Thank you for reporting this. We’ll bump a patch out for this shortly.

**Comment by NathanWalker** [comment]
published in 6.5.7 now 👍 thanks again for the report!
